fix: dialog style

Summer-Shen opened the pull request:
<!--
首先，感谢你的贡献！😄
请阅读并遵循 [TDesign 贡献指南](https://github.com/Tencent/tdesign/blob/main/docs/contributing.md)，填写以下 pull request 的信息。
PR 在维护者审核通过后会合并，谢谢！
-->

### 🤔 这个 PR 的性质是？

- [x] 日常 bug 修复
- [ ] 新特性提交
- [ ] 文档改进
- [ ] 演示代码改进
- [ ] 组件样式/交互改进
- [ ] CI/CD 改进
- [ ] 重构
- [ ] 代码风格优化
- [ ] 测试用例
- [ ] 分支合并
- [ ] 其他

### 🔗 相关 Issue

<!--
1. 描述相关需求的来源，如相关的 issue 讨论链接。
-->

https://github.com/Tencent/tdesign/issues/259

### 💡 需求背景和解决方案

<!--
1. 要解决的具体问题。
2. 列出最终的 API 实现和用法。
3. 涉及UI/交互变动需要有截图或 GIF。
-->

### 📝 更新日志

<!--
从用户角度描述具体变化，以及可能的 breaking change 和其他风险。
-->

- fix(card): 修复一些 Dialog 的样式问题，
  - 标题样式不对
  - 正文的下边距不对

- [ ] 本条 PR 不需要纳入 Changelog

### ☑️ 请求合并前的自查清单

⚠️ 请自检并全部**勾选全部选项**。⚠️

- [x] 文档已补充或无须补充
- [x] 代码演示已提供或无须提供
- [x] TypeScript 定义已补充或无须补充
- [x] Changelog 已提供或无须提供

Summer-Shen committed:
fix: dialog style

uyarn commented on style/web/components/dialog/_index.less:
```diff
@@ -45,6 +45,7 @@
   &__header {
     color: @dialog-header-text-color;
     font: @dialog-header-font;
+    font-size: @font-title-medium;
```
这个vars 是font属性使用 font-size不生效的  得改为font:

Summer-Shen commented:
@uyarn 这边 font 已经改好了，感谢 review！

honkinglin committed:
fix: 优化 drawer 动画逻辑 (#1027)

dianjie committed:
fix(submenu): 子菜单左右间距不一致 (#1029)

Wen1kang committed:
Feat: color token and switch style (#1028)

* feat: form adornment pagination slider tag-input textarea transfer size toke and fix style bug

* feat: color token and switch style

dianjie committed:
fix(head-menu): 菜单折叠 (#1016)

uyarn committed:
chore: remove dropdown item height (#1031)

honkinglin committed:
fix: 优化tooltip样式 (#1030)

* fix: 优化tooltip样式

* fix: 优化tooltip样式

uyarn committed:
chore: fix dropdown style (#1032)

Wen1kang committed:
feat: update timeline size token (#1033)

uyarn committed:
chore: add theme-desc (#1034)

dianjie committed:
feat: set firefox scrollbar track color (#1035)

TabSpace committed:
[refactor] tree 组件重构，顺带修复动画表现 (#959)

* fix(tree): label 禁止折行

* fix(tree): 完善 tree 样式细节与示例

* fix(tree): 完善 tree 组件连线样式，解决叶节点连线过短问题

* refactor(tree): tree 组件，移除无用样式，完善样式顺序格式

* fix(tree): tree 组件，连线对齐图标

* refactor(tree): tree 组件，样式变量名称改进，统一规范

* docs(tree): tree 组件样式示例，可选中示例添加长文案示例

* fix(tree): 增大图标点击区域

* docs(tree): tree 样式示例改进，自定义图标位置修正

* feat(tree): tree 组件添加嵌套布局样式

* fix(tree): tree 组件，为节点不限制高度提供可能性

* fix(tree): tree 组件添加子节点动画

* fix: tree 组件完善嵌套布局时使用的动画

* fix(tree): 动画涉及的属性，仅在动画启用时生效

* chore(tree): tree 组件，添加虚拟滚动所需部分样式

* fix: 🐛 tree 组件，解决同一个 siblings 下，前置节点插入时位置错误的问题。

* chore: 🤖 完善项目依赖

* refactor: 💡 添加事件派发器，以简化事件传播方式

* fix: 🐛 tree

tree 组件, insertAfter, insertBefore 方法，完善传入参数类型

* fix: 🐛 tree 组件动画优化

* fix: 🐛 tree 组件修正图标背景偏移的问题

* chore: 🤖 type的版本先固定 不然过不了CI的ts检测

* fix: 🐛 tree 组件添加 value 重复错误提示

* fix: 🐛 tree 组件，解决拖动样式与重构后的高度样式冲突的问题

* fix: 🐛 tree 组件，完善折行标签示例，移除嵌套模式相关样式

* fix: 🐛 tree 组件修正图标背景尺寸与位置

* fix: 🐛 tree 组件完善图标样式

* fix: 🐛 tree 组件完善动画样式

honkinglin committed:
feat: jumper 更名为 paginationMini 组件 (#1038)

* feat: jumper 更名为 paginationMini 组件

* feat: jumper 更名为 paginationMini 组件

carolin913 committed:
feat(tooltip): getpositoin support mouse (#1039)

* docs(watermark): 水印demo描述

* feat(): add tooltiplite style and demo

* feat(): add tooltiplite style and demo

* feat(): add tooltiplite style and demo

* feat(): add tooltiplite style and demo

* feat(tooltip): rate style

* feat(): 文档描述修改

* feat(): getposition add mouse for tooltip

uyarn committed:
chore: revert modify for mobile (#1040)

PsTiu committed:
fix: 修复日历组件中右上角控件栏下拉框垂直对齐的问题。 (#1042)

chaishi committed:
docs(web): table documents (#1041)

* docs(web): table

* docs(web): table

Co-authored-by: sheepluo <sheepluo@tencent.com>

HaixingOoO committed:
docs(upload):fix upload docs (#1043)

Wen1kang committed:
feat: update gray color (#1036)

honkinglin committed:
fix: 修复 dialog 动画 (#1045)

* fix: 修复 dialog 动画

* fix: 修复 dialog 动画

* fix: 修复 dialog 动画

honkinglin committed:
fix: 修复 drawer 动画效果 (#1047)

chaishi committed:
[TagInput] tag padding (#1044)

* fix(web): tag-input padding

* fix: tag padding

* fix: nowrap

* fix: margin

* style(web): remove whitespace

* style(web): remove comment

* fix: size=small

Co-authored-by: sheepluo <sheepluo@tencent.com>

pattybaby110 committed:
chore: 修改了divider文案 (#1046)

chaishi committed:
fix(web): tag-input style (#1048)

Co-authored-by: sheepluo <sheepluo@tencent.com>

Wen1kang committed:
fix: calendar style bug (#1050)

uyarn committed:
fix: menu item padding left (#1051)

honkinglin committed:
fix: 修复 collapse 翻转动画 (#1052)

honkinglin committed:
fix: 修复 button disabled 样式问题 (#1053)

chaishi committed:
fix(web): auto-width icon (#1049)

Co-authored-by: sheepluo <sheepluo@tencent.com>

uyarn committed:
chore: revert modify (#1054)

honkinglin committed:
fix: 修复 drawer 动画 (#1055)

PengYYYYY committed:
fix: table style (#1056)

chaishi committed:
feat(web): add XMLHttpRequest to events (#1064)

Co-authored-by: sheepluo <sheepluo@tencent.com>

chaishi committed:
feat: input improvement (#1065)

Co-authored-by: sheepluo <sheepluo@tencent.com>

chaishi committed:
feat(Table): empty and loading height (#1063)

* feat(Table): empty and loading height

* fix(web): empty and loading height

Co-authored-by: sheepluo <sheepluo@tencent.com>

uyarn committed:
fix(input): input inner ellipsis (#1066)

Wen1kang committed:
feat: pagination style update (#1068)

PengYYYYY committed:
fix: table line-height (#1069)

honkinglin committed:
feat: 移除 jumper (#1067)

Wen1kang committed:
feat: range input size token (#1070)

PengYYYYY committed:
fix: table date-picker (#1071)

chaishi committed:
fix(web): select-input padding (#1072)

Co-authored-by: sheepluo <sheepluo@tencent.com>

PengYYYYY committed:
fix: 修复table居中表现异常 (#1074)

* fix: 修复table居中表现异常

* fix: line-height

honkinglin committed:
fix: 修复 inputAdornment 样式 (#1075)

* fix: 修复 inputAdornment 样式

* fix: 修复 inputAdornment 样式

dianjie committed:
fix: submenu disabled color (#1062)

uyarn committed:
feat: dropdown over max height (#1076)

honkinglin committed:
fix: 修复pagination 样式问题 (#1077)

xiaosansiji committed:
Refact/menu (#1073)

* feat: menu token fix

* feat: menu UI refactor

* fix: sub menu in dark

* fix: collapsed side menu in dark mod

* fix: spelling mistake

Co-authored-by: sunzhe <sunzhe@tencent.com>

xiaosansiji committed:
fix: less vars error (#1078)

chaishi committed:
fix(web): table border style (#1079)

honkinglin committed:
fix: 修复inputAdornment 圆角问题 (#1080)

chaishi committed:
feat(web): select-input, multiple, empty, padding-right (#1082)

Co-authored-by: sheepluo <sheepluo@tencent.com>

chaishi committed:
feat(web): guide different language (#1081)

Co-authored-by: sheepluo <sheepluo@tencent.com>

honkinglin committed:
feat: 抽离公共逻辑 (#1083)

honkinglin committed:
fix: 修复datepicker tips样式问题 (#1084)

honkinglin committed:
fix: datepicker 兼容空字符串 (#1086)

honkinglin committed:
feat: 修复 tree-select 样式问题 (#1087)

zhangpaopao0609 committed:
fix: guide global config (#1085)

* fix: guide global config

* fix: add message default config

* fix: global config type

xiaosansiji committed:
chore: add menu demo style in react (#1090)

xiaosansiji committed:
fix: vue next style (#1088)

* fix: vue next style

* fix: vue next style

Commit:
chore(.gitignore): add pnpm-lock.yaml (#1089)

honkinglin committed:
fix: 修复dialog 样式细节 (#1091)

xiaosansiji committed:
fix: submenu style (#1092)

xiaosansiji committed:
fix: menu box sizing (#1094)

Co-authored-by: sunzhe <sunzhe@tencent.com>

uyarn committed:
fix(tag): select text style (#1095)

moecasts committed:
feat(tooltip-lite): mouse 模式下新增定位边界检查，保证内容在可视范围内 (#1093)

Ylushen committed:
style(image-viewer): zindex层级调整 (#1104)

* style(image-viewer): zindex层级调整

* style(image-viewer): 表头可选中

chaishi committed:
feat(web): autocomplete clear icon (#1098)

Co-authored-by: sheepluo <sheepluo@tencent.com>

sunshineYuanlei committed:
feat: 补充阿拉伯语的语言包 (#1097)

* feat: 补充阿拉伯语的语言包

* fix: lint-fix

Co-authored-by: p_pleiyuan <p_pleiyuan@tencent.com>

byq1213 committed:
fix(tag): disabled close icon (#1105)

chaishi committed:
Revert "fix(tag): disabled close icon (#1105)" (#1106)

This reverts commit 4199949e4bb9c9f010bfad7153b7a9d28631596b.

chaishi committed:
fix(web): upload.theme=single-input (#1107)

chaishi committed:
feat(web): upload support mockProgressDuration (#1108)

uyarn committed:
chore: update component number (#1109)

xiaosansiji committed:
Fix/submenu height (#1111)

* fix: submenu height

* feat: add scrollbar to popup submenu

Co-authored-by: sunzhe <sunzhe@tencent.com>

chaishi committed:
feat(web): upload.mockProgressDuration support (#1110)

honkinglin committed:
fix: 修复datepicker 禁用样式问题 (#1112)

* fix: 修复datepicker 禁用样式问题

* fix: 修复datepicker 禁用样式问题

honkinglin committed:
feat: 调整阿拉伯语言包文件名 (#1113)

dependabot[bot] committed:
chore(deps-dev): bump stylelint from 14.13.0 to 14.16.1 (#1101)

Bumps [stylelint](https://github.com/stylelint/stylelint) from 14.13.0 to 14.16.1.
- [Release notes](https://github.com/stylelint/stylelint/releases)
- [Changelog](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/stylelint/stylelint/compare/14.13.0...14.16.1)

---
updated-dependencies:
- dependency-name: stylelint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

uyarn committed:
fix(tree): label overflow (#1114)

uyarn committed:
chore: fix type by as const (#1115)

honkinglin committed:
docs: 提供获取文件 git 时间函数 (#1118)

chaishi committed:
feat(web): only empty label display: none (#1117)

Co-authored-by: sheepluo <sheepluo@tencent.com>

honkinglin committed:
fix: lint error (#1120)

chaishi committed:
[Image] ensure image in avatar won't overflow (#1119)

* feat(web): ensure image in avatar won't overflow

* docs(web): fix avatar documents

Co-authored-by: sheepluo <sheepluo@tencent.com>

Wen1kang committed:
fix: style bug (#1121)

xiaosansiji committed:
Fix/menu text color (#1122)

* fix: global text-color-brand

* fix: menu active color

* chore: remove useless menu style

* chore: remove useless html interface

Co-authored-by: sunzhe <sunzhe@tencent.com>

dianjie committed:
fix(drawer): 关闭按钮没有垂直居中 (#1037)

* fix(drawer): 关闭按钮没有垂直居中

* feat: 删除 drawer-close-btn-position-top token

chaishi committed:
feat(web): tree.label type (#1124)

Co-authored-by: sheepluo <sheepluo@tencent.com>

carolin913 committed:
image&imageviewer国际化语言配置补充 (#1125)

* docs(watermark): 水印demo描述

* feat(): add tooltiplite style and demo

* feat(): add tooltiplite style and demo

* feat(): add tooltiplite style and demo

* feat(): add tooltiplite style and demo

* feat(tooltip): rate style

* feat(): 文档描述修改

* feat(): getposition add mouse for tooltip

* feat(): add image&imageviewer locale text

* feat(): add image&imageviewer locale text

chaishi committed:
Upload Style & XHR (#1126)

* feat(web): upload xhr

* feat(web): upload

* feat(web): upload dragger-btns

* feat(web): upload buttn style

* feat(web): upload

---------

Co-authored-by: sheepluo <sheepluo@tencent.com>

yaogengzhu committed:
fix: fix dialog confirmBtnTheme default config (#1123)

Co-authored-by: yaogengzhu <4555947455@qq.com>

MrWeilian committed:
fix(table): 虚拟滚动 table 描述文案错误 (#1127)

chaishi committed:
fix(web): upload status (#1128)

* fix(web): upload status

* fix(web): upload ts error

honkinglin committed:
fix: 修复 dayjs 国际化设置问题 (#1129)

chaishi committed:
fix tree typescript type error (#1130)

* fix(web): upload status

* fix(web): upload ts error

* feat(web): update tree type

honkinglin committed:
feat: 调整字段 (#1131)

uyarn committed:
fix(tree): disabled option text color (#1132)

uyarn committed:
fix(loading): fix loading render in safari (#1138)

Wen1kang committed:
fix: steps style bug (#1139)

dependabot[bot] committed:
chore(deps-dev): bump lint-staged from 13.0.4 to 13.1.0 (#1136)

Bumps [lint-staged](https://github.com/okonet/lint-staged) from 13.0.4 to 13.1.0.
- [Release notes](https://github.com/okonet/lint-staged/releases)
- [Commits](https://github.com/okonet/lint-staged/compare/v13.0.4...v13.1.0)

---
updated-dependencies:
- dependency-name: lint-staged
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

uyarn committed:
fix: client error (#1140)

yaogengzhu committed:
fix: 修复样式覆盖问题 (#1141)

Co-authored-by: yaogengzhu <4555947455@qq.com>

uyarn committed:
feat(dropdown): optimize multiple menu overflow render (#1142)

honkinglin committed:
feat: 调整文档 (#1145)

zhangpaopao0609 committed:
feat: remove important (#1143)

uyarn committed:
feat(Select): add scroll event demo (#1146)

Simon-He95 committed:
chore: refactor & fix typo (#1147)

* chore: refactor & fix typo

* chore: update

* chore: update

* chore: fix lint & run lint:fix

honkinglin committed:
feat: 调整滚动条计算逻辑 (#1151)

* feat: 调整滚动条计算逻辑

* feat: 调整滚动条计算逻辑

PengYYYYY committed:
fix:  nuxt ssr error (#1148)

* fix: 修复ssr环境下的报错

* fix: pagination nxut error

* fix: nuxt error

* fix: nuxt error

Simon-He95 committed:
fix: lint:fix (#1149)

honkinglin committed:
fix: 修复button block 样式优先级问题 (#1152)

uyarn committed:
fix(tabs): icon style range (#1153)

honkinglin committed:
fix: 修复dialog 样式问题 (#1155)

chaishi committed:
feat(image): support .avif (#1161)

anlyyao committed:
docs(miniprogram): update design docs (#1160)

* docs: 新增mobile 设计指南 & miniprogram 设计指南文档

* Update 按钮Button.md

* Update 按钮Button.md

* Update 按钮Button.md

* Update 按钮Button.md

* Update 按钮Button.md

* Create 动作面板ActionSheet.md

* Create 悬浮按钮fab.md

* Update 悬浮按钮fab.md

* Create 分割线divider.md

* Create 抽屉drawer.md

* Rename 抽屉drawer.md to 抽屉Drawer.md

* Rename 悬浮按钮fab.md to 悬浮按钮Fab.md

* Rename 分割线divider.md to 分割线Divider.md

* Create 底部标签栏TabBar.md

* Create 选项卡Tabs.md

* Create 图片Image.md

* Update and rename 底部标签栏TabBar.md to 标签栏TabBar.md

* Create 图片预览ImageViewer.md

* Create 单选框Radio.md

* Create 多选框Checkbox.md

* Create 滑动选择器Slider.md

* Update 动作面板ActionSheet.md

* Update 动作面板ActionSheet.md

* Update 动作面板ActionSheet.md

* Update 动作面板ActionSheet.md

* Update 动作面板ActionSheet.md

* Update 悬浮按钮Fab.md

* Update 分割线Divider.md

* Update 动作面板ActionSheet.md

* Update 动作面板ActionSheet.md

* Update 按钮Button.md

* Update 按钮Button.md

* Update 动作面板ActionSheet.md

* Update 悬浮按钮Fab.md

* Update 悬浮按钮Fab.md

* Update 悬浮按钮Fab.md

* Update 悬浮按钮Fab.md

* Update 按钮Button.md

* docs: update design docs

* Update 按钮Button.md

* Update 按钮Button.md

* Update 分割线Divider.md

* Update 悬浮按钮Fab.md

* Update 动作面板ActionSheet.md

* Rename 分割线Divider.md to Divider.md

* Update Divider.md

* Rename 动作面板ActionSheet.md to ActionSheet.md

* Rename 单选框Radio.md to Radio.md

* Update and rename 图片Image.md to Image.md

* Update Image.md

* Update Image.md

* Update 多选框Checkbox.md

* Rename 多选框Checkbox.md to Checkbox.md

* Update 图片预览ImageViewer.md

* Rename 图片预览ImageViewer.md to ImageViewer.md

* Update Radio.md

* Rename 悬浮按钮Fab.md to Fab.md

* Rename 按钮Button.md to Button.md

* Update and rename 滑动选择器Slider.md to Slider.md

* Update 标签栏TabBar.md

* Rename 标签栏TabBar.md to TabBar.md

* Update ImageViewer.md

* Update ImageViewer.md

* Update and rename 选项卡Tabs.md to Tabs.md

* Update Tabs.md

* Update Tabs.md

* Update and rename 抽屉Drawer.md to Drawer.md

* Update Button.md

* Update Checkbox.md

* Update Drawer.md

* Update Fab.md

* Update Image.md

* Update ImageViewer.md

* Update Radio.md

* Update Slider.md

* Update TabBar.md

* Update Tabs.md

* Update ActionSheet.md

* Update Slider.md

* Update Tabs.md

* Update ImageViewer.md

* Create Dialog.md

* Update Button.md

* Create PullDownRefresh.md

* Update PullDownRefresh.md

* Update PullDownRefresh.md

* docs: update design docs

* Update PullDownRefresh.md

* Update Checkbox.md

* Update Dialog.md

---------

Co-authored-by: HQ-Lin <a154571896@gmail.com>
Co-authored-by: caixuan29 <96420688+caixuan29@users.noreply.github.com>
Co-authored-by: pageee1021 <94840745+pageee1021@users.noreply.github.com>
Co-authored-by: eugegewu <103838357+eugenewoody@users.noreply.github.com>

tiny-dust committed:
fix: 修复menu-item设置了href属性和icon插槽后，菜单折叠导致的icon变小以及展开时，点击右侧空白处不能跳转的问题 (#1158)

uyarn committed:
fix(dialog): default config btn theme (#1164)

TabSpace committed:
feat: tree 组件增加虚拟滚动能力 (#1159)

* fix(tree): label 禁止折行

* fix(tree): 完善 tree 样式细节与示例

* fix(tree): 完善 tree 组件连线样式，解决叶节点连线过短问题

* refactor(tree): tree 组件，移除无用样式，完善样式顺序格式

* fix(tree): tree 组件，连线对齐图标

* refactor(tree): tree 组件，样式变量名称改进，统一规范

* docs(tree): tree 组件样式示例，可选中示例添加长文案示例

* fix(tree): 增大图标点击区域

* docs(tree): tree 样式示例改进，自定义图标位置修正

* feat(tree): tree 组件添加嵌套布局样式

* fix(tree): tree 组件，为节点不限制高度提供可能性

* fix(tree): tree 组件添加子节点动画

* fix: tree 组件完善嵌套布局时使用的动画

* fix(tree): 动画涉及的属性，仅在动画启用时生效

* chore(tree): tree 组件，添加虚拟滚动所需部分样式

* fix: 🐛 tree 组件，解决同一个 siblings 下，前置节点插入时位置错误的问题。

* chore: 🤖 完善项目依赖

* refactor: 💡 添加事件派发器，以简化事件传播方式

* fix: 🐛 tree

tree 组件, insertAfter, insertBefore 方法，完善传入参数类型

* fix: 🐛 tree 组件动画优化

* fix: 🐛 tree 组件修正图标背景偏移的问题

* chore: 🤖 type的版本先固定 不然过不了CI的ts检测

* fix: 🐛 tree 组件添加 value 重复错误提示

* fix: 🐛 tree 组件，解决拖动样式与重构后的高度样式冲突的问题

* fix: 🐛 tree 组件，完善折行标签示例，移除嵌套模式相关样式

* fix: 🐛 tree 组件修正图标背景尺寸与位置

* fix: 🐛 tree 组件完善图标样式

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件完善虚拟滚动样式

* fix: 🐛 tree 组件完善虚拟滚动样式，添加滚动中样式

* fix: 🐛 tree 节点添加独立的私有 id ，专用于渲染

* fix: 🐛 tree 组件添加节点私有 id 映射

* fix: 🐛 tree 组件，解决动画样式导致虚拟滚动卡顿的问题

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件，收纳所有动画效果到动画 class 之下管理

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件，改进默认样式，支持滚动

* fix: 🐛 tree 组件完善虚拟滚动样式

* fix: 🐛 tree 组件修正单元测试方案，不直接审查唯一 key 的值。移除冗余判断条件，准备改进过滤方案。

* fix: 🐛 完善类型指定

* fix: 🐛 tree 组件，改进对 allowFoldNodeOnFilter 的支持，过滤节点时不影响折叠状态

* fix: 🐛 tree 组件，解决手风琴效果丢失的问题

* fix: 🐛 tree 组件，解决父组件状态未能正确更新的问题

* docs: ✏️ 添加虚拟滚动示例

caixuan29 committed:
Update PullDownRefresh.md (#1163)

chaishi committed:
[Upload] 自定义上传方法，支持一个请求上传返回多个文件的数据回显 (#1165)

* feat(image): support .avif

* feat(web): upload support multiple files in one request

---------

Co-authored-by: sheepluo <sheepluo@tencent.com>

uyarn committed:
chore: fix type (#1166)

duanbaosheng committed:
feat: fix textarea tysle (#1157)

* feat: fix textarea tysle

* feat: textarea style

* feat: textarea style

* feat: textarea style

* feat: modify textarea style

* feat: modify textarea styles

chaishi committed:
fix: upload failed (#1167)

* fix: upload

* fix: lint error

---------

Co-authored-by: sheepluo <sheepluo@tencent.com>

honkinglin committed:
fix: 修复tree多余滚动条 (#1168)

TabSpace committed:
[fix] tree 组件，完善虚拟滚动样式，完善过滤状态的样式 (#1177)

* fix(tree): label 禁止折行

* fix(tree): 完善 tree 样式细节与示例

* fix(tree): 完善 tree 组件连线样式，解决叶节点连线过短问题

* refactor(tree): tree 组件，移除无用样式，完善样式顺序格式

* fix(tree): tree 组件，连线对齐图标

* refactor(tree): tree 组件，样式变量名称改进，统一规范

* docs(tree): tree 组件样式示例，可选中示例添加长文案示例

* fix(tree): 增大图标点击区域

* docs(tree): tree 样式示例改进，自定义图标位置修正

* feat(tree): tree 组件添加嵌套布局样式

* fix(tree): tree 组件，为节点不限制高度提供可能性

* fix(tree): tree 组件添加子节点动画

* fix: tree 组件完善嵌套布局时使用的动画

* fix(tree): 动画涉及的属性，仅在动画启用时生效

* chore(tree): tree 组件，添加虚拟滚动所需部分样式

* fix: 🐛 tree 组件，解决同一个 siblings 下，前置节点插入时位置错误的问题。

* chore: 🤖 完善项目依赖

* refactor: 💡 添加事件派发器，以简化事件传播方式

* fix: 🐛 tree

tree 组件, insertAfter, insertBefore 方法，完善传入参数类型

* fix: 🐛 tree 组件动画优化

* fix: 🐛 tree 组件修正图标背景偏移的问题

* chore: 🤖 type的版本先固定 不然过不了CI的ts检测

* fix: 🐛 tree 组件添加 value 重复错误提示

* fix: 🐛 tree 组件，解决拖动样式与重构后的高度样式冲突的问题

* fix: 🐛 tree 组件，完善折行标签示例，移除嵌套模式相关样式

* fix: 🐛 tree 组件修正图标背景尺寸与位置

* fix: 🐛 tree 组件完善图标样式

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件完善虚拟滚动样式

* fix: 🐛 tree 组件完善虚拟滚动样式，添加滚动中样式

* fix: 🐛 tree 节点添加独立的私有 id ，专用于渲染

* fix: 🐛 tree 组件添加节点私有 id 映射

* fix: 🐛 tree 组件，解决动画样式导致虚拟滚动卡顿的问题

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件，收纳所有动画效果到动画 class 之下管理

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件，改进默认样式，支持滚动

* fix: 🐛 tree 组件完善虚拟滚动样式

* fix: 🐛 tree 组件修正单元测试方案，不直接审查唯一 key 的值。移除冗余判断条件，准备改进过滤方案。

* fix: 🐛 完善类型指定

* fix: 🐛 tree 组件，改进对 allowFoldNodeOnFilter 的支持，过滤节点时不影响折叠状态

* fix: 🐛 tree 组件，解决手风琴效果丢失的问题

* fix: 🐛 tree 组件，解决父组件状态未能正确更新的问题

* docs: ✏️ 添加虚拟滚动示例

* fix: 🐛 tree 组件，虚拟滚动延迟加载模式，使用 class t-tree__lazyload

* fix: 🐛 tree 组件，解决过滤允许折叠时，锁定的路径节点样式未变更的问题

anlyyao committed:
style: update css token (#1169)

* style: update css token

* fix: fix lint

* fix: fix cr

chaishi committed:
docs(web): update image docs (#1178)

meiqi502 committed:
feat: add back-top component (#1175)

* add back-top style

* feat: add back-top style

* feat: add back-top styles

* feat: update back-top.md

---------

Co-authored-by: mickeylin <mickeylin@tencent.com>

TabSpace committed:
fix: tree 组件，解决激活态未能正确更新视图的问题 (#1180)

* fix(tree): label 禁止折行

* fix(tree): 完善 tree 样式细节与示例

* fix(tree): 完善 tree 组件连线样式，解决叶节点连线过短问题

* refactor(tree): tree 组件，移除无用样式，完善样式顺序格式

* fix(tree): tree 组件，连线对齐图标

* refactor(tree): tree 组件，样式变量名称改进，统一规范

* docs(tree): tree 组件样式示例，可选中示例添加长文案示例

* fix(tree): 增大图标点击区域

* docs(tree): tree 样式示例改进，自定义图标位置修正

* feat(tree): tree 组件添加嵌套布局样式

* fix(tree): tree 组件，为节点不限制高度提供可能性

* fix(tree): tree 组件添加子节点动画

* fix: tree 组件完善嵌套布局时使用的动画

* fix(tree): 动画涉及的属性，仅在动画启用时生效

* chore(tree): tree 组件，添加虚拟滚动所需部分样式

* fix: 🐛 tree 组件，解决同一个 siblings 下，前置节点插入时位置错误的问题。

* chore: 🤖 完善项目依赖

* refactor: 💡 添加事件派发器，以简化事件传播方式

* fix: 🐛 tree

tree 组件, insertAfter, insertBefore 方法，完善传入参数类型

* fix: 🐛 tree 组件动画优化

* fix: 🐛 tree 组件修正图标背景偏移的问题

* chore: 🤖 type的版本先固定 不然过不了CI的ts检测

* fix: 🐛 tree 组件添加 value 重复错误提示

* fix: 🐛 tree 组件，解决拖动样式与重构后的高度样式冲突的问题

* fix: 🐛 tree 组件，完善折行标签示例，移除嵌套模式相关样式

* fix: 🐛 tree 组件修正图标背景尺寸与位置

* fix: 🐛 tree 组件完善图标样式

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件完善虚拟滚动样式

* fix: 🐛 tree 组件完善虚拟滚动样式，添加滚动中样式

* fix: 🐛 tree 节点添加独立的私有 id ，专用于渲染

* fix: 🐛 tree 组件添加节点私有 id 映射

* fix: 🐛 tree 组件，解决动画样式导致虚拟滚动卡顿的问题

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件，收纳所有动画效果到动画 class 之下管理

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件，改进默认样式，支持滚动

* fix: 🐛 tree 组件完善虚拟滚动样式

* fix: 🐛 tree 组件修正单元测试方案，不直接审查唯一 key 的值。移除冗余判断条件，准备改进过滤方案。

* fix: 🐛 完善类型指定

* fix: 🐛 tree 组件，改进对 allowFoldNodeOnFilter 的支持，过滤节点时不影响折叠状态

* fix: 🐛 tree 组件，解决手风琴效果丢失的问题

* fix: 🐛 tree 组件，解决父组件状态未能正确更新的问题

* docs: ✏️ 添加虚拟滚动示例

* fix: 🐛 tree 组件，虚拟滚动延迟加载模式，使用 class t-tree__lazyload

* fix: 🐛 tree 组件，解决过滤允许折叠时，锁定的路径节点样式未变更的问题

* fix: 🐛 tree 组件，解决 active 状态切换后，旧激活节点状态未更新的问题

TabSpace committed:
fix: 解决节点插入自身引发死循环的问题 (#1181)

* fix(tree): label 禁止折行

* fix(tree): 完善 tree 样式细节与示例

* fix(tree): 完善 tree 组件连线样式，解决叶节点连线过短问题

* refactor(tree): tree 组件，移除无用样式，完善样式顺序格式

* fix(tree): tree 组件，连线对齐图标

* refactor(tree): tree 组件，样式变量名称改进，统一规范

* docs(tree): tree 组件样式示例，可选中示例添加长文案示例

* fix(tree): 增大图标点击区域

* docs(tree): tree 样式示例改进，自定义图标位置修正

* feat(tree): tree 组件添加嵌套布局样式

* fix(tree): tree 组件，为节点不限制高度提供可能性

* fix(tree): tree 组件添加子节点动画

* fix: tree 组件完善嵌套布局时使用的动画

* fix(tree): 动画涉及的属性，仅在动画启用时生效

* chore(tree): tree 组件，添加虚拟滚动所需部分样式

* fix: 🐛 tree 组件，解决同一个 siblings 下，前置节点插入时位置错误的问题。

* chore: 🤖 完善项目依赖

* refactor: 💡 添加事件派发器，以简化事件传播方式

* fix: 🐛 tree

tree 组件, insertAfter, insertBefore 方法，完善传入参数类型

* fix: 🐛 tree 组件动画优化

* fix: 🐛 tree 组件修正图标背景偏移的问题

* chore: 🤖 type的版本先固定 不然过不了CI的ts检测

* fix: 🐛 tree 组件添加 value 重复错误提示

* fix: 🐛 tree 组件，解决拖动样式与重构后的高度样式冲突的问题

* fix: 🐛 tree 组件，完善折行标签示例，移除嵌套模式相关样式

* fix: 🐛 tree 组件修正图标背景尺寸与位置

* fix: 🐛 tree 组件完善图标样式

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件完善虚拟滚动样式

* fix: 🐛 tree 组件完善虚拟滚动样式，添加滚动中样式

* fix: 🐛 tree 节点添加独立的私有 id ，专用于渲染

* fix: 🐛 tree 组件添加节点私有 id 映射

* fix: 🐛 tree 组件，解决动画样式导致虚拟滚动卡顿的问题

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件，收纳所有动画效果到动画 class 之下管理

* fix: 🐛 tree 组件完善动画样式

* fix: 🐛 tree 组件，改进默认样式，支持滚动

* fix: 🐛 tree 组件完善虚拟滚动样式

* fix: 🐛 tree 组件修正单元测试方案，不直接审查唯一 key 的值。移除冗余判断条件，准备改进过滤方案。

* fix: 🐛 完善类型指定

* fix: 🐛 tree 组件，改进对 allowFoldNodeOnFilter 的支持，过滤节点时不影响折叠状态

* fix: 🐛 tree 组件，解决手风琴效果丢失的问题

* fix: 🐛 tree 组件，解决父组件状态未能正确更新的问题

* docs: ✏️ 添加虚拟滚动示例

* fix: 🐛 tree 组件，虚拟滚动延迟加载模式，使用 class t-tree__lazyload

* fix: 🐛 tree 组件，解决过滤允许折叠时，锁定的路径节点样式未变更的问题

* fix: 🐛 tree 组件，解决 active 状态切换后，旧激活节点状态未更新的问题

* fix: 🐛 tree 组件，不允许节点插入到自身，并给出提示信息

anlyyao committed:
fix: restoring wrongly modified content (#1184)

anlyyao committed:
style: primary renamed brand (#1185)

honkinglin committed:
feat: datepicker 支持defaultTime (#1186)

* feat: datepicker 支持defaultTime

* feat: datepicker 支持defaultTime

* feat: datepicker 支持defaultTime

KMethod committed:
fix: 修复dialog 垂直居中 (#1187)

Co-authored-by: methodchen <kmthod@foxmail.com>

yc910920 committed:
文档更新 (#1182)

chaishi committed:
feat(web): table, disabled fixed column draggable (#1179)

* feat(web): table, disbaled fixed column draggable

* docs(web): update table docs

* doc(web): table

---------

Co-authored-by: sheepluo <sheepluo@tencent.com>

puppetsheep committed:
feat(locale): 新增繁體中文(台灣地區)語言支持 (#1189)

honkinglin committed:
fix: 修复日语韩语星期翻译问题 (#1190)

xiaosansiji committed:
Feat/optimize eslint config (#1191)

* feat: add hex color rule

* fix: correct color token variables

* feat: add disallowed named color rule

* fix: color vars lint error

---------

Co-authored-by: sunzhe <sunzhe@tencent.com>

honkinglin committed:
fix: 修复时间格式化问题 (#1194)

* fix: 修复时间格式化问题

* fix: 修复时间格式化问题

honkinglin committed:
fix: 修复类型检测问题 (#1195)

honkinglin committed:
fix: 修复类型检测问题 (#1196)

uyarn committed:
chore: fix typo (#1193)

huangpiqiao committed:
fix: 修复存在表格左边冻结列，往右边滚动时，hover表头出现边框后，表头冻结列和表格内容冻结列没有对齐问题 (#1197)

* fix: 修复表格左边冻结列左右滚动时，hover表头出现边框后，表头冻结列和表格内容冻结列没有对齐问题

* fix(table): 修复格式

---------

Co-authored-by: pechelhuang <pechelhuang@tencent.com>

anlyyao committed:
style(Stepper): update style for mobile (#1198)

anlyyao committed:
feat: add api_v2 folder (#1199)

uyarn committed:
feat: add formatThousandths utils (#1200)

tiny-dust committed:
fix: 补充transparent变量，修复textarea获取焦点后的样式问题(#2509) (#1176)

* fix: 补充transparent变量，修复textarea获取焦点后的样式问题(#2509)

* fix(textarea): remove css var

ZekunWu committed:
feature(sticky-tool): add sticky-tool component style (#1183)

* feat: 部分样式

* feat: 尝试百分比尺寸

* feat: 完善css

* feat: 样式调整

* fix: 删除wxss文件

* fix: lint

* fix: 改用全局变量

* feat: 修改布局方式

---------

Co-authored-by: yiranwu <yiranwu@tencent.com>

anlyyao committed:
feat: add v2 folder (#1201)

oljc committed:
feat(locale): 新增俄语、意大利语国际化语言支持 (#1202)

* feat(loacle): 新增俄语(俄罗斯)语言支持文件

* feat(loacle): 新增意大利语(意大利)语言支持文件

* fix: lint use singlequote

caixuan29 committed:
Update PullDownRefresh.md (#1162)

uyarn committed:
docs: update docs for i18n (#1203)

LeeJim committed:
feat(radio): update the radio style on mobile (#1204)

* feat(radio): update the radio style on mobile

* fix: resolve lint errors

* fix: resolve lint errors

* fix: resolve lint errors

* fix: update stylelint config

* fix: update style

* fix: resolve lint errors

* docs: update api

* fix: using correct token

LeeJim committed:
feat(switch): update style for mobile (#1206)

* feat(switch): update style for mobile

* fix: resolve lint errors

anlyyao committed:
fix: resolve the file path error (#1208)

Wen1kang committed:
feat:add color token (#1207)

Co-authored-by: kadenzhong <kadenzhong@tencent.com>

anlyyao committed:
docs(Textarea): update docs (#1209)

LeeJim committed:
feat(loading): using new style (#1211)

* feat(loading): using new style

* fix: resolve lint errors

uyarn committed:
fix(tree): fix empty node (#1213)

meiqi502 committed:
feat: update backTop style and doc (#1214)

jarmywang committed:
feat(checkbox): update new style and doc (#1212)

* feat(checkbox): update new style and doc

* fix: add stylelint for webkit-box

LeeJim committed:
feat(cell-group): add style for v2 (#1215)

* feat(cell-group): add style for v2

* fix: resolve lint errors

anlyyao committed:
fix(Textarea): update class name (#1216)

chaishi committed:
feat(web): update table documents (#1219)

Co-authored-by: sheepluo <sheepluo@tencent.com>

oljc committed:
feat(tatistic): Add statistic component style (#1218)

* feat(tatistic): Add statistic component style

* fix: lint fix

* fix: Use the canonical font token

ZekunWu committed:
Feature/web/sticky tool (#1221)

* feat: 部分样式

* feat: 尝试百分比尺寸

* feat: 完善css

* feat: 样式调整

* fix: 删除wxss文件

* fix: lint

* fix: 改用全局变量

* feat: 修改布局方式

* feat: 暂存尝试联动backtop

* feat: 初始化sticky-tool demo文档

* chore: update description

---------

Co-authored-by: yiranwu <yiranwu@tencent.com>
Co-authored-by: yuyang <uyarnchen@gmail.com>

uyarn committed:
chore: update backtop docs (#1222)

uyarn committed:
chore: fix title (#1223)

LeeJim committed:
fix: update picker style (#1224)

uyarn committed:
fix: format time with format (#1227)

uyarn committed:
fix: format time for range (#1228)

Wen1kang committed:
fix: alert icon and close padding style bug (#1229)

* fix: alert icon and close padding style bug

* chore: fix lint

---------

Co-authored-by: kadenzhong <kadenzhong@tencent.com>
Co-authored-by: yuyang <uyarnchen@gmail.com>

scshsy committed:
fix(image-viewer): esm下less会编译失败，需要转义less文件中min函数 (#1225)

Co-authored-by: gomezshui <gomezshui@tencent.com>

uyarn committed:
chore: loading conic background backup (#1230)

anlyyao committed:
feat: add empty docs (#1231)

GolderBrother committed:
docs: 增加 breadcrumb、dropdown、menu、pagination 组件的api英文文档 (#1232)

* docs: 增加 breadcrumb、dropdown、menu、pagination 组件的api英文文档

* docs: menu 组件英文文档翻译内容调整

---------

Co-authored-by: James <jamesezhang@tencent.com>

GolderBrother committed:
docs: 增加 drawer、message、notification、popu等组件英文文档 (#1235)

Co-authored-by: James <jamesezhang@tencent.com>

ZekunWu committed:
Feature/web/sticky tool (#1241)

* feat: 部分样式

* feat: 尝试百分比尺寸

* feat: 完善css

* feat: 样式调整

* fix: 删除wxss文件

* fix: lint

* fix: 改用全局变量

* feat: 修改布局方式

* feat: 暂存尝试联动backtop

* feat: 初始化sticky-tool demo文档

* chore: update description

* fix: lint:fix

* fix: lint:fix

* fix: 颜色修复

---------

Co-authored-by: yiranwu <yiranwu@tencent.com>
Co-authored-by: yuyang <uyarnchen@gmail.com>

anlyyao committed:
docs(Search): update docs (#1226)

ZekunWu committed:
Feature/web/sticky tool (#1244)

* feat: 部分样式

* feat: 尝试百分比尺寸

* feat: 完善css

* feat: 样式调整

* fix: 删除wxss文件

* fix: lint

* fix: 改用全局变量

* feat: 修改布局方式

* feat: 暂存尝试联动backtop

* feat: 初始化sticky-tool demo文档

* chore: update description

* fix: lint:fix

* fix: lint:fix

* fix: 颜色修复

* feat: 添加hover动画

* feat: 完善动画

---------

Co-authored-by: yiranwu <yiranwu@tencent.com>
Co-authored-by: yuyang <uyarnchen@gmail.com>

anlyyao committed:
docs(Cascader): update docs (#1220)

anlyyao committed:
style(Search): update style (#1245)

anlyyao committed:
docs(Sticky): update docs (#1242)

anlyyao committed:
docs(Skeleton): update docs (#1243)

ccccpj committed:
feat: input v2 & border (#1217)

* feat: input v2 & border

* fix: lint

* fix: lint empty line

TingShine committed:
docs(image): update image demo (#995)

* docs: update image demo

* docs: update image demo docs

* docs(image): update image docs

dependabot[bot] committed:
chore(deps-dev): bump lint-staged from 13.1.4 to 13.2.0 (#1236)

Bumps [lint-staged](https://github.com/okonet/lint-staged) from 13.1.4 to 13.2.0.
- [Release notes](https://github.com/okonet/lint-staged/releases)
- [Commits](https://github.com/okonet/lint-staged/commits/v13.2.0)

---
updated-dependencies:
- dependency-name: lint-staged
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

LeeJim committed:
fix: update loading style and image docs (#1248)

Ericleungs committed:
fix(InputNumber): float add integer error (#1251)

xiaosansiji committed:
docs: fix md format problem (#1253)

Co-authored-by: sunzhe <sunzhe@tencent.com>

GolderBrother committed:
docs: 增加 cascader、checkbox、date-picker等输入组件的英文文档 (#1249)

* docs: 增加 cascader、checkbox、date-picker等输入组件的英文文档

* docs:  md 文档开头的格式规范，description字段前面添加换行，后面添加空格

---------

Co-authored-by: James <jamesezhang@tencent.com>

ccccpj committed:
feat: update input md (#1252)

* feat: update input md

* fix: 纠正

anlyyao committed:
feat: update design docs for miniprogram (#1254)

* Create link.md

* Create Indexes.md

* Create cascader.md

* Update cascader.md

* Create avatar.md

* Update cascader.md

* Create swiper.md

* Create navbar.md

* Update navbar.md

* Update Indexes.md

* Update avatar.md

* Update link.md

* Update cascader.md

* Update navbar.md

* Update swiper.md

* Update Drawer.md

* Update Drawer.md

* Update Drawer.md

* Update Drawer.md

* Update swiper.md

* Update avatar.md

* Update cascader.md

* Update navbar.md

* Update Indexes.md

* Update Drawer.md

* Update swiper.md

* Update Drawer.md

* Create Sliderbar.md

* Update Sliderbar.md

* Create Steps.md

* Update link.md

* Update Indexes.md

* Update navbar.md

* Update cascader.md

* Update avatar.md

* Update swiper.md

* Create Calendar.md

* Update avatar.md

* Update link.md

* Update Drawer.md

* Update Indexes.md

* Update navbar.md

* Update navbar.md

* Update cascader.md

* Update swiper.md

* Create DateTimePicker.md

* Update DateTimePicker.md

* Create Input.md

* Create Picker.md

* Create Rate.md

* Create Search.md

* Create Switch.md

* Create Textarea.md

* Create Upload.md

* Create Badge.md

* Create CountDown.md

* Create Empty.md

* Create Progress.md

* Create Result.md

* Create Loading.md

* Create Message.md

* Create Swipecell.md

* Create Toast.md

* Rename Sliderbar.md to Sidebar.md

* feat: update design docs

---------

Co-authored-by: caixuan29 <96420688+caixuan29@users.noreply.github.com>
Co-authored-by: eugegewu <103838357+eugenewoody@users.noreply.github.com>
Co-authored-by: pageee1021 <94840745+pageee1021@users.noreply.github.com>

Wen1kang committed:
fix:tab style bug (#1255)

uyarn committed:
fix: remove extra right style for autowidth inputnumber (#1258)

Ericleungs committed:
Fix/date picker wrong last week (#1257)

* fix(datePicker): last week of year swapped into first error

* docs(datePicker): add comment for codes

ccccpj committed:
feat: update cursor-pointer (#1259)

* feat: update cursor-pointer

* fix: fix lint

---------

Co-authored-by: anlyyao <anly_yaw@163.com>

jarmywang committed:
Update Badge.md (#1260)

xiaosansiji committed:
chore: remove useless code (#1262)

Co-authored-by: sunzhe <sunzhe@tencent.com>

PDieE committed:
fix: 修正在部分情况下timeline-item__dot为重置box-sizing而导致的样式错误 (#1264)

uyarn committed:
fix(popconfirm): inner text font weight (#1265)

Commit:
fix(input): override edge password input default style (#1261)

LeeJim committed:
fix(grid): update mobile style (#1263)

* fix(grid): update mobile style

* fix: resolve lint errors

* fix: resolve lint errors

* fix: resolve lint errors

fengchunqi committed:
refactor(rate): rate popover (#1233)

* refactor(rate): rate popover

* docs: rate docs

xiaosansiji committed:
docs: fix md render error (#1266)

Co-authored-by: sunzhe <sunzhe@tencent.com>

LeeJim committed:
fix(upload): using new style (#1269)

yusuf-yz committed:
refactor(message): new message (#1267)

brianzhang committed:
Refactor/link (#1270)

* link Mobile Component

* create link doc

* fix: lint fix

tangzixuan committed:
Refractor dialog (#1250)

anlyyao committed:
feat: update design for miniprogram (#1271)

* Create link.md

* Create Indexes.md

* Create cascader.md

* Update cascader.md

* Create avatar.md

* Update cascader.md

* Create swiper.md

* Create navbar.md

* Update navbar.md

* Update Indexes.md

* Update avatar.md

* Update link.md

* Update cascader.md

* Update navbar.md

* Update swiper.md

* Update Drawer.md

* Update Drawer.md

* Update Drawer.md

* Update Drawer.md

* Update swiper.md

* Update avatar.md

* Update cascader.md

* Update navbar.md

* Update Indexes.md

* Update Drawer.md

* Update swiper.md

* Update Drawer.md

* Create Sliderbar.md

* Update Sliderbar.md

* Create Steps.md

* Update link.md

* Update Indexes.md

* Update navbar.md

* Update cascader.md

* Update avatar.md

* Update swiper.md

* Create Calendar.md

* Update avatar.md

* Update link.md

* Update Drawer.md

* Update Indexes.md

* Update navbar.md

* Update navbar.md

* Update cascader.md

* Update swiper.md

* Create DateTimePicker.md

* Update DateTimePicker.md

* Create Input.md

* Create Picker.md

* Create Rate.md

* Create Search.md

* Create Switch.md

* Create Textarea.md

* Create Upload.md

* Create Badge.md

* Create CountDown.md

* Create Empty.md

* Create Progress.md

* Create Result.md

* Create Loading.md

* Create Message.md

* Create Swipecell.md

* Create Toast.md

* Rename Sliderbar.md to Sidebar.md

* Update and rename Sidebar.md to SideBar.md

* Update SideBar.md

* Update Steps.md

* Update and rename Calendar.md to Calender.md

* Update and rename Calender.md to Calendar.md

* Update DateTimePicker.md

* Update Input.md

* Update Picker.md

* Update Picker.md

* Update Rate.md

* Update Search.md

* Update Switch.md

* Update Textarea.md

* Update Switch.md

* Update Progress.md

* Update Message.md

* Update Swipecell.md

* Update Toast.md

* Update Calendar.md

* Update Toast.md

* Update Swipecell.md

* Update Swipecell.md

* Update Message.md

* Update Progress.md

* Update Empty.md

* Update CountDown.md

* Update Calendar.md

* Update Calendar.md

* Update Calendar.md

* Update DateTimePicker.md

* Update DateTimePicker.md

* Update Input.md

* Update Picker.md

* Update Rate.md

* Update Rate.md

* Update Rate.md

* Update Rate.md

* Update Badge.md

* Update PullDownRefresh.md

* Update Drawer.md

* Update Drawer.md

* Update PullDownRefresh.md

* Update Calendar.md

* Update DateTimePicker.md

* Update Input.md

* Update Badge.md

* Update Empty.md

* Update Progress.md

* Update Message.md

* Update and rename Swipecell.md to SwipeCell.md

* Update Toast.md

* Update Checkbox.md

* Update Radio.md

* Update Slider.md

* Update Slider.md

* Update Button.md

* Update Divider.md

* Update Divider.md

* Update Fab.md

* Update link.md

* Update Indexes.md

* Update navbar.md

* Update TabBar.md

* Update Tabs.md

* Update cascader.md

* Update avatar.md

* Update Image.md

* Update ImageViewer.md

* Update swiper.md

* Update ActionSheet.md

* Update Dialog.md

* Update Loading.md

* Update Loading.md

* Update SideBar.md

* Update Button.md

* Update Divider.md

* Update Button.md

* Update Divider.md

* Update TabBar.md

* Update SideBar.md

* Update SideBar.md

* Update Steps.md

* Update Steps.md

* Update Steps.md

* Update ActionSheet.md

* Update Drawer.md

* Update Badge.md

* Update Calendar.md

* Update CountDown.md

* Update DateTimePicker.md

* Update Dialog.md

* Update Input.md

* Update Loading.md

* Update Message.md

* Update Picker.md

* Update Progress.md

* Update PullDownRefresh.md

* Update Rate.md

* Update Result.md

* Update Search.md

* Update Slider.md

* Update Steps.md

* Update SwipeCell.md

* Update TabBar.md

* Update Tabs.md

* Update Textarea.md

* Update Toast.md

* Update Upload.md

* Update cascader.md

* Update Switch.md

* Update Upload.md

* Update Upload.md

* Update Textarea.md

* Update Toast.md

* Update Toast.md

* Update Dialog.md

* Update Result.md

* Update Search.md

* Update Button.md

* Update TabBar.md

* Update Textarea.md

* Update Upload.md

* Update Steps.md

* Update Indexes.md

* feat: update design

---------

Co-authored-by: caixuan29 <96420688+caixuan29@users.noreply.github.com>
Co-authored-by: eugegewu <103838357+eugenewoody@users.noreply.github.com>
Co-authored-by: pageee1021 <94840745+pageee1021@users.noreply.github.com>

brianzhang committed:
Refactor/link (#1272)

* link Mobile Component

* create link doc

* fix: lint fix

* fix: link hover style

LeeJim committed:
feat: add docs for footer (#1274)

brianzhang committed:
Refactor/link (#1278)

* link Mobile Component

* create link doc

* fix: lint fix

* fix: link hover style

* fix: update link style

* feat: create link style

tangzixuan committed:
fix: refractor dialog style (#1277)

anlyyao committed:
feat: add form form mobile (#1276)

* feat: add form form mobile

* fix: fix cr

* feat(Upload): enrich css variable

* fix: fix cr

jarmywang committed:
feat: divider改造 (#1281)

* Update Divider.md

* fix(divider): add end line(#1281)

LeeJim committed:
fix(overlay): update style (#1282)

anlyyao committed:
fix(Dialog): resolve close-icon style (#1283)

anlyyao committed:
fix: fix design docs file name error (#1285)

LeeJim committed:
fix(popup): update style for mobile (#1286)

mistakers committed:
feat: toast改造 (#1279)

* feat: toast改造

* feat: toast-md更新

---------

Co-authored-by: thewang <thewang@tencent.com>

anlyyao committed:
fix(Toast): update style (#1287)

* fix(Toast): update style

* Update _index.less

uyarn committed:
chore: update theme generator desc (#1288)

uyarn committed:
fix(ColorPikcer): toggle color format in gradient mode (#1289)

vapourma committed:
翻译issue (#1293)

LeeJim committed:
fix(button): update style on mobile (#1294)

* fix(button): update style on mobile

* fix: resolve lint errors

xiaosansiji committed:
docs: add loading en doc (#1295)

Co-authored-by: sunzhe <sunzhe@tencent.com>

LeeJim committed:
docs: update overview for mobile (#1297)

anlyyao committed:
docs(Slider): update docs (#1290)

yaogengzhu committed:
refactor: new tag (#1296)

Co-authored-by: yaogengzhu <yaogengzhu@tencent.com>

yaogengzhu committed:
docs(Tag): update docs  (#1298)

* doc: 与小程序端保持一致

* fix: update docs

---------

Co-authored-by: yaogengzhu <yaogengzhu@tencent.com>

anlyyao committed:
feat: update docs (#1299)

LeeJim committed:
docs: update fab (#1300)

LeeJim committed:
fix(image): resolve vertical align (#1301)

LeeJim committed:
docs: update cell (#1302)

Ericleungs committed:
fix(date-picker): 修复在时间戳模式下panel会显示Invalid value的问题 (#1268)

* fix(date-picker): 修复在时间戳模式下panel会显示Invalid value的问题

* feat(date-picker): 调整了代码结构，简化部分if判断环节

honkinglin committed:
fix: 修复类型问题 (#1303)

anlyyao committed:
docs: update design for miniprogram (#1304)

* Create link.md

* Create Indexes.md

* Create cascader.md

* Update cascader.md

* Create avatar.md

* Update cascader.md

* Create swiper.md

* Create navbar.md

* Update navbar.md

* Update Indexes.md

* Update avatar.md

* Update link.md

* Update cascader.md

* Update navbar.md

* Update swiper.md

* Update Drawer.md

* Update Drawer.md

* Update Drawer.md

* Update Drawer.md

* Update swiper.md

* Update avatar.md

* Update cascader.md

* Update navbar.md

* Update Indexes.md

* Update Drawer.md

* Update swiper.md

* Update Drawer.md

* Create Sliderbar.md

* Update Sliderbar.md

* Create Steps.md

* Update link.md

* Update Indexes.md

* Update navbar.md

* Update cascader.md

* Update avatar.md

* Update swiper.md

* Create Calendar.md

* Update avatar.md

* Update link.md

* Update Drawer.md

* Update Indexes.md

* Update navbar.md

* Update navbar.md

* Update cascader.md

* Update swiper.md

* Create DateTimePicker.md

* Update DateTimePicker.md

* Create Input.md

* Create Picker.md

* Create Rate.md

* Create Search.md

* Create Switch.md

* Create Textarea.md

* Create Upload.md

* Create Badge.md

* Create CountDown.md

* Create Empty.md

* Create Progress.md

* Create Result.md

* Create Loading.md

* Create Message.md

* Create Swipecell.md

* Create Toast.md

* Rename Sliderbar.md to Sidebar.md

* Update and rename Sidebar.md to SideBar.md

* Update SideBar.md

* Update Steps.md

* Update and rename Calendar.md to Calender.md

* Update and rename Calender.md to Calendar.md

* Update DateTimePicker.md

* Update Input.md

* Update Picker.md

* Update Picker.md

* Update Rate.md

* Update Search.md

* Update Switch.md

* Update Textarea.md

* Update Switch.md

* Update Progress.md

* Update Message.md

* Update Swipecell.md

* Update Toast.md

* Update Calendar.md

* Update Toast.md

* Update Swipecell.md

* Update Swipecell.md

* Update Message.md

* Update Progress.md

* Update Empty.md

* Update CountDown.md

* Update Calendar.md

* Update Calendar.md

* Update Calendar.md

* Update DateTimePicker.md

* Update DateTimePicker.md

* Update Input.md

* Update Picker.md

* Update Rate.md

* Update Rate.md

* Update Rate.md

* Update Rate.md

* Update Badge.md

* Update PullDownRefresh.md

* Update Drawer.md

* Update Drawer.md

* Update PullDownRefresh.md

* Update Calendar.md

* Update DateTimePicker.md

* Update Input.md

* Update Badge.md

* Update Empty.md

* Update Progress.md

* Update Message.md

* Update and rename Swipecell.md to SwipeCell.md

* Update Toast.md

* Update Checkbox.md

* Update Radio.md

* Update Slider.md

* Update Slider.md

* Update Button.md

* Update Divider.md

* Update Divider.md

* Update Fab.md

* Update link.md

* Update Indexes.md

* Update navbar.md

* Update TabBar.md

* Update Tabs.md

* Update cascader.md

* Update avatar.md

* Update Image.md

* Update ImageViewer.md

* Update swiper.md

* Update ActionSheet.md

* Update Dialog.md

* Update Loading.md

* Update Loading.md

* Update SideBar.md

* Update Button.md

* Update Divider.md

* Update Button.md

* Update Divider.md

* Update TabBar.md

* Update SideBar.md

* Update SideBar.md

* Update Steps.md

* Update Steps.md

* Update Steps.md

* Update ActionSheet.md

* Update Drawer.md

* Update Badge.md

* Update Calendar.md

* Update CountDown.md

* Update DateTimePicker.md

* Update Dialog.md

* Update Input.md

* Update Loading.md

* Update Message.md

* Update Picker.md

* Update Progress.md

* Update PullDownRefresh.md

* Update Rate.md

* Update Result.md

* Update Search.md

* Update Slider.md

* Update Steps.md

* Update SwipeCell.md

* Update TabBar.md

* Update Tabs.md

* Update Textarea.md

* Update Toast.md

* Update Upload.md

* Update cascader.md

* Update Switch.md

* Update Upload.md

* Update Upload.md

* Update Textarea.md

* Update Toast.md

* Update Toast.md

* Update Dialog.md

* Update Result.md

* Update Search.md

* Update Button.md

* Update TabBar.md

* Update Textarea.md

* Update Upload.md

* Update Steps.md

* Update Indexes.md

* Create DropdownMenu.md

* Create Footer.md

* Create Cell.md

* Create Popup.md

* Create TreeSelect.md

* Create Collapse.md

* Create Grid.md

* Update DropdownMenu.md

* Update Popup.md

* Update TreeSelect.md

* Update Collapse.md

* Update Cell.md

* Update DropdownMenu.md

* Update Footer.md

* Update Cell.md

* Update Cell.md

* Update Cell.md

* Update Popup.md

* Update TreeSelect.md

* Update Collapse.md

* Update Grid.md

* Update Footer.md

* Update Cell.md

* Update Grid.md

* Update Collapse.md

* Update DropdownMenu.md

* Update Footer.md

* Update Cell.md

* Update Popup.md

* Update TreeSelect.md

* Update Collapse.md

* Update Grid.md

* Create Tag.md

* Update Tag.md

* Update Collapse.md

* Update Grid.md

* Update Tag.md

* Update Tag.md

* feat: update design

---------

Co-authored-by: caixuan29 <96420688+caixuan29@users.noreply.github.com>
Co-authored-by: eugegewu <103838357+eugenewoody@users.noreply.github.com>
Co-authored-by: pageee1021 <94840745+pageee1021@users.noreply.github.com>

LeeJim committed:
fix(cell): resolve icon align (#1306)

LeeJim committed:
docs(dialog): update api (#1307)

Commit:
fix(menu): disabled attribute does not take effect (#1273)

uyarn committed:
fix(switch): use border-radius token replace switch radius (#1311)

fengchunqi committed:
Refator/rate (#1310)

* refactor(rate): rate popover

* docs: rate docs

* refactor(rate): rate tips

uyarn committed:
chore: fix type (#1312)

* chore: fix type

* chore: fix type

nined9 committed:
refactor(BackTop): update docs and _index.less (#1309)

* refactor(BackTop): update docs and _index.less

* fix: use hairline

uyarn committed:
chore: fix docs (#1313)

* chore: fix docs

* chore: format

xiaosansiji commented:
@Summer-Shen 麻烦同步下上游库 develop 的代码，处理下冲突

uyarn committed:
fix(switch): use border-radius token replace switch radius (#1316)

Summer-Shen committed:
Merge branch 'fix/dialog-style' of github.com:Summer-Shen/tdesign-common into fix/dialog-style

Summer-Shen commented:
@xiaosansiji 您好！这边 rebase 好了，感谢！
